steps/dependency-check/1.0.0/step.yml

```diff
@@ -0,0 +1,154 @@
+title: Dependency Check
+summary: A wrapper for the OWASP dependency check tool
+description: Statically analyses a codebase and its dependencies for potential security
+  vulnerabilities
+website: https://github.com/alpheroltd/bitrise-step-dependency-check
+source_code_url: https://github.com/alpheroltd/bitrise-step-dependency-check
+support_url: https://github.com/alpheroltd/bitrise-step-dependency-check/issues
+published_at: 2022-11-24T10:16:51.375141+13:00
+source:
+  git: https://github.com/alpheroltd/bitrise-step-dependency-check.git
+  commit: 24c466c0e62c363b2cf6bd2378b5b4656adb51f3
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/alpheroltd/bitrise-step-dependency-check
+deps:
+  brew:
```

Comment:
Is it supposed to run on Linux VMs too? If so, you can define an `apt` dependency here (assuming the tool is available in APT)

https://devcenter.bitrise.io/en/builds/installing-tools-during-a-build.html#installing-tools-by-declaring-deps-in-the-bitrise-yml-file